chapters/p07_44_46.md

```diff
@@ -0,0 +1,3 @@
+> # Part 7: Debugging inference algorithms
+
+# Phần 7: Gỡ lỗi các Thuật toán suy luận
```

Comment:
@ngcthuong @sonvx @naml3i có đề xuất gì về cách dịch từ "inference" không? Và cả cụm "end-to-end" ở phần 8 bên dưới nữa.

Reply:
Khó phết, end-to-end literally là "từ đầu (này) đến đầu (khác)" = "từ đầu đến cuối". Nếu không ngại "nôm" quá thì có thể dùng luôn "từ đầu đến cuối". Theo thesaurus thì end-to-end có synonym là "throughout" = `xuyên suốt`. 
Vậy có thể dịch "an end-to-end learning system" là "một hệ thống học xuyên suốt"? Hay cầu kì hơn, "hệ thống học xuyên suốt đầu cuối"?

Reply:
@naml3i @duythanhvn từ "end-to-end" mình đã dịch là "đầu-cuối" kèm giải nghĩa. Từ ngày có nghĩa là "đầu đến cuối", dấu gạch ngang vừa có nghĩa là "đến", vừa giữ được dấu gạch ngang trong bản tiếng Anh. 

Tạm thời mình chấp nhận cách dịch này để dịch tiếp các chương.